tests/Functional/Schema/MySQLSchemaManagerTest.php

```diff
@@ -544,6 +544,44 @@ public function testEnsureTableOptionsAreReflectedInMetadata(): void
         ], $onlineTable->getOption('create_options'));
     }
 
+    public function testCreatedCompositeForeignKeySchemaDoesNotDifferAfterCreation(): void
+    {
+        $this->connection->executeStatement('DROP TABLE IF EXISTS test_table_metadata');
+
+        $sql = <<<'SQL'
+CREATE TABLE test_table_foreign_key_referenced (
```

Comment:
Please create the table using the `Schema` API and move the test to the base class. This scenario is valid for all supported platforms.

Reply:
@morozov So I should create it like I do with the `test_table_with_foreign_key`  table correct?

Reply:
Yes.